test/evergreen.yml

```diff
@@ -1688,6 +1582,93 @@ variables:
           <<: *configure_flags_with_builtins
       - func: "run workgen test"
 
+#########################################################################################
+#    Stress tests that validate disagg behavior in leader, follower, and switch modes.  #
+#########################################################################################
+
+  - &format-stress-test-disagg-leader-palm
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          format_args: disagg.page_log=palm disagg.mode=leader runs.rows=100000 runs.ops=100000
+          times: 5
+
+  - &format-stress-test-disagg-leader-verify
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          # FIXME-WT-15619 and FIXME-WT-15618 Remove this task when disagg verification is working reliably with deltas.
+          format_args: disagg.mode=leader ops.verify=1 disagg.leaf_page_delta=0 disagg.internal_page_delta=0
```

Comment:
I see WT-15619 and WT-15618 being close to merged, maybe it's worth waiting until then?

Reply:
@raviprakashgiri29 do you think once these 2 tickets are closed we will be able to run verify with deltas?

Reply:
@Siddhartha8899 Yes, after [WT-15619](https://jira.mongodb.org/browse/WT-15619) and [WT-15618](https://jira.mongodb.org/browse/WT-15618) the verify can be enabled. WT-15618 I merged today and WT-15619 is planned to close in this sprint.

Reply:
Thanks @raviprakashgiri29 I see there are many fix me for these 2 tickets in the code, is it possible to ensure they are all sorted it [WT-15619](https://jira.mongodb.org/browse/WT-15619)? I will likely merge this tomorrow.

Reply:
@Siddhartha8899 Cleanup for [WT-15618](https://jira.mongodb.org/browse/WT-15618) is complete; all relevant comments were removed within that ticket. The comments related to [WT-15619](https://jira.mongodb.org/browse/WT-15619) will be addressed during its implementation, but this does not block merging this pull request.